lib/rake/ci.rake

```diff
@@ -64,6 +70,10 @@ namespace :ci do
     if CI::Utils.tagged?(RUN_ALL_TESTS_TAG)
       ChatClient.log "Commit message: '#{CI::Utils.git_commit_message}' contains [#{RUN_ALL_TESTS_TAG}], force-running all tests."
       RakeUtils.rake_stream_output 'test:all'
+    elsif CI::Utils.tagged?(SKIP_PEGASUS_CONTENT)
+      # Because we only test with SKIP_PEGASUS_CONTENT periodically, we want to run all tests regardless of which files changed.
+      ChatClient.log "Commit message: '#{CI::Utils.git_commit_message}' contains [#{SKIP_PEGASUS_CONTENT}], force-running all tests."
+      RakeUtils.rake_stream_output 'test:all'
```

Comment:
I'm having trouble thinking through this bit clearly, but my system will be /always/ running with SKIP_PEGASUS_CONTENT=true, and this logic will probably be something I have to hack out or work around: I'm not sure I want to have --run-all-tests forcibly added when I run in this mode! Maybe print a warning instead? Or is there a use case this fixes?